src/applications/mhv-medical-records/containers/VitalDetails.jsx

```diff
@@ -334,7 +334,10 @@ Provider notes: ${vital.notes}\n\n`,
         </ul>
 
         {/* print view start */}
-        <h1 className="vads-u-font-size--h1 vads-u-margin-bottom--1 print-only">
+        <h1
+          className="vads-u-font-size--h1 vads-u-margin-bottom--1 print-only"
+          data-dd-privacy="mask"
+        >
```

Comment:
This is the "print-only" `<h1>`. You can leave this one, but we need one on the `<h1>` up near line 250 as well. That's the one that gets displayed on the web page.